modules/Bio/EnsEMBL/Compara/PipeConfig/LoadMembers_conf.pm

```diff
@@ -516,14 +516,14 @@ sub core_pipeline_analyses {
             %hc_analysis_params,
             -flow_into          => {
                 '2->A' => WHEN( '#load_uniprot_members#' => 'save_uniprot_release_date' ),
-                'A->1' => [ 'dc_members_enums' ],
+                'A->1' => [ 'dc_sequences_and_enums' ],
```

Comment:
Since we have the list of DCs in the analysis, what do you think about calling the analysis in a more generic way?
```suggestion
                'A->1' => [ 'datachecks' ],
```
I think adding what was being checked in the analysis name was done because the HCs didn't always have a comprehensive name/task.